docs/deployment/rds-s3/README.md

```diff
@@ -53,7 +53,38 @@ eksctl create cluster \
 --managed
 ```
 
-4. Create S3 Bucket
+4. Create an OIDC provider for your cluster  
+**Important :**  
+You must make sure you have an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for your cluster and that it was added from `eksctl` >= `0.56` or if you already have an OIDC provider in place, then you must make sure you have the tag `alpha.eksctl.io/cluster-name` with the cluster name as its value. If you don't have the tag, you can add it via the AWS Console by navigating to IAM->Identity providers->Your OIDC->Tags.
+
+## 2.0 Setup RDS, S3 and configure Secrets
+There are two ways to create the RDS and S3 resources before you deploy the Kubeflow manifests. Either use the automated python script we have provided by following the steps in section 2.1 or fall back on the manual setup steps as in section 2.2 below - 
+
+### 2.1 **Option 1: Automated Setup**
+This setup performs all the manual steps in an automated fashion.  
+The script takes care of creating the S3 bucket, creating the S3 secrets using the secrets manager, setting up the RDS database and creating the RDS secret using the secrets manager. It also edits the required configuration files for Kubeflow pipeline to be properly configured for the RDS database during Kubeflow installation.  
+The script also handles cases where the resources already exist in which case it will simply skips the step.
+  
+Note : The script will **not** delete any resource therefore if a secret already exist or if a database with the same name already exist or if the S3 bucket already exists for instance, it will skip those steps. This was done in order to prevent unwanted results such as accidental deletion. For instance, if a database with the same name already exists, the script will skip the database creation setup. If it's expected in your scenario, then perhaps this is fine for you, if you simply forgot to change the database name used for creation then this gives you the chance to retry the script with the proper value. See `python auto-rds-s3-setup.py --help` for the list of parameters as well as their default values.
```

Comment:
If resource already exist, I believe your script will use it. In that case we should call it out here.

```
If resources (bucket, RDS, secrets) already exists, the script will use it as it is.
```

Reply:
Great suggestion, I reworked the doc to follow your suggestion.